shared/src/test/scala/ElementTests.scala

```diff
@@ -354,6 +354,14 @@ class ElementTests extends VyxalTests:
     }
   }
 
+  describe("Element H") {
+    testMulti("H")(
+      List[VAny](VNum(69420)) -> "10F2C",
+      List[VAny](VNum(0)) -> "0",
+      List[VAny](VNum(89)) -> "59"
```

Comment:
Might want to add some reverse tests here (string => num)

Reply:
Can't believe I forgot those